### PR TITLE
Feature/retrieve library and populate cards

### LINKED ISSUE #41: Retrieve library from database and populate library page

### PR DESCRIPTION
**Pull request by JonHarrison**

### CODE REVIEW

File: src/pages/Library/index.js [hunk 1]
```diff
@@ -1,79 +1,112 @@
...
+    const booksSnapshot = await getDocs(collectionRef)
+    let items = [];
+    if (booksSnapshot.docs.length > 0) {
+      booksSnapshot.forEach((doc) => {
+        console.log('[Library] booksRef - doc ', doc.id, doc.data())
+        items.push(doc.data().item)
+      })
+    }
+
+    // const querySnapshot = await getDocs(collectionRef)
```
Comment (ansarraja) [inline src/pages/Library/index.js]: can the commented out code be removed

File: src/pages/Library/index.js [hunk 2]
```diff
@@ -1,79 +1,112 @@
...
+    // const books = querySnapshot.docs.map((d) => ({ id: d.id, ...d.data() }))
```
Comment (ansarraja) [inline src/pages/Library/index.js]: same, can this be removed if not required

File: src/pages/Library/style.css [hunk 3]
```diff
@@ -1,14 +1,24 @@
 .App-library {
+    overflow-y: auto;
 }
 
-.libraryCard{
+.App-library-book-list {
```
Comment (ansarraja) [inline src/pages/Library/style.css:5]: can commented out style be removed if not required

**Review by ansarraja — COMMENTED** [review]
Hi Jon I added few comments to remove redundant (commented out) code

File: src/pages/Library/index.js (hunk 1 above)
Comment (JonHarrison) [reply src/pages/Library/index.js]: Done

File: src/pages/Library/index.js (hunk 2 above)
Comment (JonHarrison) [reply src/pages/Library/index.js]: Done

File: src/pages/Library/style.css (hunk 3 above)
Comment (JonHarrison) [reply src/pages/Library/style.css:5]: This page needs styling, I'd rather leave it in there in case it needs to be put back

**Review by ansarraja — APPROVED** [review]

**Pull request merged**